Fix wxTextCtrl::SetSelection to match the docs:
"If both parameters are equal to -1 all text in the control is selected"

Signed-off-by: Anders Larsen <al@alarsen.net>
---
 src/univ/textctrl.cpp |    4 +++-
 1 files changed, 3 insertions(+), 1 deletion(-)

Index: b/src/univ/textctrl.cpp
===================================================================
--- a/src/univ/textctrl.cpp
+++ b/src/univ/textctrl.cpp
@@ -1434,8 +1434,10 @@ wxString wxTextCtrl::GetSelectionText() 
 void wxTextCtrl::SetSelection(wxTextPos from, wxTextPos to)
 {
     // selecting till -1 is the same as selecting to the end
-    if ( to == -1 && from != -1 )
+    if ( to == -1 )
     {
+        if ( from == -1 )
+            from = 0;
         to = GetLastPosition();
     }
 

 	  	 
